.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contribution guidelines
+
+All contributors should read and understand this guideline before their contribution. Maintainers will reject any contribution not following this guideline as necessary.
+
+This guideline is agreed upon by all maintainers.
+
+## Outside contributors
+
+You are encouraged to:
+
+- Report any bugs encountered in using Aya.
+  - Please include a minimal reproduction and the commit hash you are using to run the reproduction.
+  - Please include the error report.
+- Ask any questions about Aya in the discussion area.
+
+These are currently not accepted:
+
+- PRs, because Aya is still in rapid development.
+- Feature suggestions.
+
+Your responsibility:
+
+- Unethical behaviors not adhering to [the Code of Conduct](CODE_OF_CONDUCT.md) is not allowed and will be taken proper action upon by maintainers.
+
+If you are interested in contributing more and becoming a member of the organization, please contact the maintainers.
+
+## Members of the organization
+
+You are encouraged to:
+
+- Open issues about the feature planning, implementation and other aspects of the Aya prover.
+- Open bug fixing PRs.
+- Open feature adding PRs when the addition of the feature is already agreed on by the community.
+
+You are not allowed to:
+
+- Push to the main branch, except for a few exceptions.Merging your PR into the main branch is wholly handled by the @bors bot.
+- Force-push to the main branch. This is strictly prohibited. No exception.
+- Perform other destructive and/or irreversible actions on the organization or repository settings without the maintainers' consent.
+
+Your responsibility:
```

Comment:
One more thing: base yourself on the latest main branch if possible and make sure you look at the CI test results. Make sure you understand the purpose of using CI and address the problems it reports.